dcplib/checksumming_io/s3_etag.py

```diff
@@ -7,17 +7,23 @@
 
 
 class S3Etag:
-    etag_stride = 64 * 1024 * 1024
 
-    def __init__(self):
+    default_chunk_size = 64 * 1024 * 1024
+
+    def __init__(self, chunk_size):
         self._etag_bytes = 0
         self._etag_parts = []
         self._etag_hasher = hashlib.md5()
+        self._chunk_size = chunk_size or self.default_chunk_size
```

Comment:
this should just be `self._chuck_size = chunk_size`

Reply:
✅